python/hail/matrixtable.py

```diff
@@ -445,12 +445,12 @@ def get_globals(self):
 
         Returns
         -------
-        :class:`.Struct`
+        :class:`.StructExpression`
             Global values.
         """
         if self._globals is None:
             self._globals = self.global_schema._convert_to_py(self._jvds.globals())
-        return self._globals
+        return to_expr(self._globals)
```

Comment:
I'm very confused. How is this different from `ds.globals`? Do we need this now?

Reply:
is it because we can't eval something with a source and no axes? We should fix that.

Reply:
I don't know what this was about.  I deleted it.